run_mpas_analysis

```diff
@@ -92,6 +92,11 @@ def build_analysis_list(config, refConfig):  # {{{
                                             refConfig))
     analyses.append(ocean.ClimatologyMapSSH(config, oceanClimatolgyTask,
                                             refConfig))
+    for bgcvar in config.getExpression('climatologyMapBGC', 'variables'):
```

Comment:
I'd like to avoid having this level of task-specific code (reading config options, etc.) in run_mpas_analysis.  I see two good options:
1. You could put this loop inside of `ClimatologyMapBGC.__init__()` so that there would only be one task with a very large number of subtasks that does all the remapping and plotting for all fields.  This would be my slight preference.  See my new `ClimatologyMapSchmidtko` for an example.
2. You could move this bit of code to a function at the top of `climatology_map_bgc.py` and just call that function from here.